numpy/f2py/crackfortran.py

```diff
@@ -903,6 +905,16 @@ def _is_intent_callback(vdecl):
     return 0
 
 
+def _resolvetypedefpattern(line):
+    line = markouterparen(line)
+    m1 = typedefpattern.match(line)
+    if m1:
+        attributes = m1.group('attributes')
+        attributes = [a.strip().lower() for a in attributes.split(',')] if attributes is not None else []
```

Comment:
Use:
```python
attributes = [a.lower() for a in attributes.split(',')] if attributes else []
```

Reply:
Removed the strip() since already done when removing the spaces. Written as you suggested.